charts/pega/README.md

```diff
@@ -841,20 +841,20 @@ certificates:
 ```
 ## Deploying Hazelcast in Client Server Mode 
 
-The values.yaml section for `hazelcast` is used for configuration of a Client Server form of deployment for Hazelcast. Pega Platform supports the client-server model for the Hazelcast service, which provides cluster communication and distributes Pega Platform features across Kubernetes PODs. This deployment model introduces 
-independent scalability for both servers and clients in Pega Platform. **For Kubernetes based deployments Client server is the default and the recommended form of deployment.**
+The values.yaml section for `hazelcast` is used for configuration of a Client Server form of deployment for Hazelcast. 
+Pega Platform deployments using 8.6 and later, supports the client-server model with the Hazelcast service. This deployment model introduces 
+independent scalability for both servers and clients in Pega Platform. 
+**For Pega Platform deployments using 8.6 and later, this client server is the default and the recommended form of deployment.**
 
-Deploying the Pega provided `platform/clustering-service` docker image which contains the Hazelcast clustering service image inside it, 
+Deploying the Pega provided `platform/clustering-service` Docker image which contains the Hazelcast clustering service image inside it, 
 starts a cluster of hazelcast server nodes. For the discovery of hazelcast members in the cluster, a plugin provided by Hazelcast, namely Hazelcast-Kubernetes Plugin is used. 
-Out of the two discovery strategies that the latter plugin provides - Kubernetes API and DNS Lookup, we make use of DNS lookup for resolving the IPs of PODs running hazelcast.
+Out of the two discovery strategies that the latter plugin provides - Kubernetes API and DNS Lookup, the client server model with Hazelcast uses DNS lookup to resolve the IP addressing of PODs running Hazelcast.
```

Comment:
tweak>>
 Kubernetes API and DNS Lookup, the client-server model with Hazelcast uses DNS lookup to resolve the IP addressing of PODs running Hazelcast.